lib/routes/routeBackbeat.js

```diff
@@ -58,8 +58,16 @@ PUT /_/backbeat/<bucket name>/<object key>/data
 */
 
 function putData(request, response, bucketInfo, objMd, log, callback) {
-    const canonicalID = request.headers['x-scal-canonicalId'];
-    const contentMd5 = request.headers['content-md5'];
+    const canonicalID = request.headers['x-scal-canonical-id'];
+    if (canonicalID === undefined) {
+        log.error('bad request: missing x-scal-canonical-id header');
```

Comment:
maybe it's a backbeat specific route and such cannot be considered coming from a user?